editor/settings/editor_settings.cpp

```diff
@@ -363,6 +363,28 @@ void EditorSettings::_set_initialized() {
 	initialized = true;
 }
 
+static LocalVector<String> _get_skipped_locales() {
+	// Skip locales if Text server lack required features.
+	LocalVector<String> locales_to_skip;
+	if (!TS->has_feature(TextServer::FEATURE_BIDI_LAYOUT) || !TS->has_feature(TextServer::FEATURE_SHAPING)) {
+		locales_to_skip.push_back("ar"); // Arabic
```

Comment:
Oh yeah, probably each one of these could end with a period as per style guide?
```suggestion
		locales_to_skip.push_back("ar"); // Arabic.
```